packages/nimble-components/src/anchor-tabs/specs/README.md

```diff
@@ -0,0 +1,204 @@
+
+Tabs do not participate in forms, so there is no ControlValueAccessor.
+
+**[routerLink] and [routerLinkActive] Support**
+
+We will follow the same approach previously used for the BreadcrumbItem, Anchor, and AnchorButton:
+
+-   We will create a directive that extends `RouterLinkWithHref`, and its selector will be `nimble-anchor-tab[nimbleRouterLink]`. The directive will define `@Input nimbleRouterLink` (which sets `routerLink`). This is a small change for clients which we will document (consistent with BreadcrumbItem, Anchor, and AnchorButton), and other routerLink attributes can still be used as-is:
+    ```html
+    <nimble-anchor-tab
+        nimbleRouterLink="/customapp"
+        [queryParams]="{debug: true}"
+        [state]="{tracingId: 123}"
+    >
+        Custom App Page
+    </nimble-anchor-tab>
+    ```
+-   A separate directive will be created to disable use of the `routerLink` attribute on `nimble-anchor-tab` elements.
+
+### Blazor integration
+
+Blazor wrappers will be created for both `nimble-anchor-tabs` and `nimble-anchor-tab`. There are no special routing considerations/mechanisms for Blazor.
+
+### Visual Appearance
+
+[XD Design](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/b2aa2c0c-03b7-4571-8e0d-de88baf0814b/)
+
+The design for navigation tabs will be the same as for standard tabs. The standard Nimble tabs do not (yet) have specific support for icons or alert indicators, so we will (initially) not implement those for navigation tabs either.
+
+In addition to indicating which tab is active, navigation tabs must also separately indicate which tab has keyboard focus. I.e. you can arrow through tabs without activating those tabs, and there must be a visual indication of which tab has keyboard focus. This is not specified in the linked design document. TODO
+
+---
+
+## Implementation
+
+As long as there is at least one non-disabled tab in the tabs group, we will have an active tab. Upon connecting `nimble-navigation-tabs` to the DOM, we will activate (i.e. navigate to) the tab specified as initially active. If no tab was specified as initially active, we will default to the first non-disabled tab.
+
+Each `nimble-navigation-tab` must have an id so that `nimble-navigation-tabs`'s `activeid` has a valid value. If the user does not specify an id for a tab, we will generate one using FAST's `uniqueId()` utility function.
+
+If the active tab becomes disabled, it will remain active. We will not automatically activate a different tab.
+
+When the tabs group gains or loses focus we must set or remove the `focused` class from one of the child tabs. To do this, we will add `onfocus` and `onblur` event handlers to the tabs group.
+
+
+### States
+
+#### Keyboard focused state:
+When tabbing to the `nimble-navigation-tabs`, it gets keyboard focus. To indicate this, the active tab (which is already underlined) gets a second, thinner underline.
+
+### Accessibility
```

Comment:
I'd recommend checking out https://www.w3.org/WAI/ARIA/apg/patterns/tabpanel/#keyboard-interaction-21 for a full set of keyboard interactions.

Reply:
Thanks. In the accessibility section, I added interactions for the Home and End keys, which I had missed. Also, added that the Space key activates the tab.